src/pages/resolve/pages/admin/Admin.vue

```diff
@@ -0,0 +1,192 @@
+            </q-stepper>
+        </div>
+        <div class="form-wrapper">
+            <q-dialog v-model="form">
+                <init-election-modal
+                    v-if="formType === 'initelection'"
+                    :close="closeModal"
+                />
+                <begin-voting-modal
+                    v-if="formType === 'beginvoting'"
+                    :close="closeModal"
+                />
+            </q-dialog>
+        </div>
+    </div>
+</template>
+
+<script>
+import InitElectionModal from '../../components/InitElectionModal.vue';
+import BeginVotingModal from '../../components/BeginVotingModal.vue';
+import { mapGetters } from 'vuex';
+
+export default {
+    components: {
+        InitElectionModal,
+        BeginVotingModal
+    },
+    data() {
+        return {
+            form: null,
+            formType: null
+        };
+    },
+    methods: {
+        closeModal() {
+            this.form = null;
+        },
+        async endElection() {
+            const endElectionActions = [
+                {
+                    account: process.env.ARB_CONTRACT,
+                    name: 'endelection',
+                    data: {}
+                }
+            ];
+            try {
+                await this.$store.$api.signTransaction(endElectionActions);
+            } catch (err) {
+                console.log('endElection error: ', err);
+            }
+        }
+    },
+    computed: {
+        ...mapGetters({
+            arbSeatsAvailable: 'resolve/arbSeatsAvailable',
+            isResolveStoresAvailable: 'resolve/isResolveStoresAvailable'
+        }),
+        electionStatus() {
+            const resolve = this.$store.state.resolve;
+            if (resolve && resolve.config && resolve.elections) {
+                const { current_election_id } = resolve;
+                const currentElection = resolve.elections.find(
+                    e => e.id === current_election_id
+                );
+                if (!currentElection) return null;
+                const {
+                    status,
+                    end_voting_ts,
+                    end_add_candidates_ts
+                } = currentElection;
+                if (status === 1) {
+                    const endAddCandidateUnixTimestamp = new Date(
+                        `${end_add_candidates_ts}Z`
+                    ).getTime();
+                    const rightNow = new Date().getTime();
+                    if (endAddCandidateUnixTimestamp > rightNow) {
+                        return 'candidate-registration';
+                    } else {
+                        return 'election-ready';
+                    }
+                }
+                if (status === 2) {
+                    // see if voting period has ended
+                    const endVotingUnixTimestamp = new Date(
+                        `${end_voting_ts}Z`
+                    ).getTime();
+                    const rightNow = new Date().getTime();
```

Comment:
`rightNow` can be assigned once in parent scope, can refactor 161-163 and 149-151, could just move to a utility method to re-use throughout.